node.gyp

```diff
@@ -284,6 +284,9 @@
                     ],
                   },
                   'conditions': [
+                    ['OS=="win"', {
+                      'sources': ['<(SHARED_INTERMEDIATE_DIR)/openssl.def'],
```

Comment:
I guess you can also do this? Did this build as an executable? Otherwise the commit I linked also works.